graph/test/integration/Performance/PerfTester.cpp

```diff
@@ -0,0 +1,186 @@
+/**
+ * File: PerfTester.cpp
+ * License: Part of the MetaCG project. Licensed under BSD 3 clause license. See LICENSE.txt file at
+ * https://github.com/tudasc/metacg/LICENSE.txt
+ */
+
+#include "Callgraph.h"
+#include "io/MCGReader.h"
+#include "io/MCGWriter.h"
+
+#include <chrono>
+#include <functional>
+#include <random>
+
+using namespace metacg;
+
+class Timer {
+ public:
+  using clock = std::chrono::high_resolution_clock;
+  using tp = clock::time_point;
+  using duration = clock::duration;
```

Comment:
```suggestion
  using Clock = std::chrono::high_resolution_clock;
  using TP = clock::time_point;
  using Duration = clock::duration;
```
Don't know if this makes sense, but I would treat them as "types from MCG" and those are capitalized.